src/awkward/contents/listarray.py

```diff
@@ -1395,15 +1395,15 @@ def _recursively_apply(
             and self._backend.nplike.known_data
             and self._starts.length != 0
         ):
-            startsmin = self._backend.index_nplike.min(self._starts.data).item()
+            startsmin = self._backend.index_nplike.min(self._starts.data)
```

Comment:
I assume we don't need `.item()` here because of the subsequent `self._starts.data - startsmin`, which CuPy understands as a `1D` array and a `0D` array being subtracted from one another